index.html

```diff
@@ -1777,7 +1777,10 @@ <h2 id="Collection">Backbone.Collection</h2>
     <p id="Collection-remove">
       <b class="header">remove</b><code>collection.remove(models, [options])</code>
       <br />
-      Remove a model (or an array of models) from the collection, and returns them.
+      Remove a model (or an array of models) from the collection, and returns
+      them. <tt>models</tt> (or each of its elements if an array) can be any
```

Comment:
Sorry -- that should've said it'd be confusing referring *just* to get. How about enumerating the options but also adding in the link?